src/contracts/constants.ts

```diff
@@ -19,3 +19,6 @@ export const ETH_FOX_POOL_CONTRACT_ADDRESS = '0x470e8de2ebaef52014a47cb5e6af8688
 // Exported as a string literal for contract address discrimination purposes
 export const UNISWAP_V2_ROUTER_02_CONTRACT_ADDRESS =
   '0x7a250d5630b4cf539739df2c5dacb4c659f2488d' as const
+
+export const IDLE_PROXY_1_CONTRACT_ADDRESS = '0xaca81583840B1bf2dDF6CDe824ada250C1936B4D'
```

Comment:
Checksummed is on purpose here, which is how unchained is returning us transfers `from` and `to` addresses.

With that being said, and not related to this PR per se, we should probably be consistent WRT using either checksummed/non-checksummed addresses everywhere, and doing the right conversions in-place, we've been burned by this before (precisely in the Tx history slice lookups)

Reply:
Good comment, maybe add a concise version to the code that this is checksummed as it's what unchained returns.

Reply:
05d16675cbf1c31cb156a0d97cba30b3d6b05032